rc-events-service/src/main/scala/com/epam/random_coffee/events/services/EventService.scala

```diff
@@ -1,16 +1,22 @@
 package com.epam.random_coffee.events.services
 
-import com.epam.random_coffee.events.model.{ Event, EventId }
+import com.epam.random_coffee.events.model.{ Event, EventId, RandomCoffeeEvent }
 
 import scala.concurrent.Future
 
 trait EventService {
 
-  def create(eventName: String): Future[Event]
+  def create(
+    name: String,
+    description: String,
+    eventDate: String,
+    author: String
+  ): Future[RandomCoffeeEvent]
```

Comment:
Same as before, you can use full RandomCoffeeEvent instead of taking each field from class